cmd/commands/runtime.go

```diff
@@ -248,25 +248,27 @@ func RunRuntimeList() error {
 	}
 
 	for _, rt := range runtimes {
-		status := "N/A"
-		namespace := "N/A"
-		name := "N/A"
-		cluster := "N/A"
-		version := "N/A"
-		if rt.Status != nil {
-			status = rt.Status.String()
-		}
-		if rt.Namespace != nil {
-			namespace = *rt.Namespace
-		}
-		if rt.ObjectMeta != nil && rt.ObjectMeta.Name != nil {
-			name = *rt.ObjectMeta.Name
-		}
-		if rt.Cluster != nil {
-			cluster = *rt.Cluster
-		}
-		if rt.Version != nil {
-			version = *rt.Version
+		status := rt.HealthStatus
+		namespace := rt.Metadata.Namespace
+		name := rt.Metadata.Name
+		cluster := rt.Cluster
+		version := rt.Version
+
+		// TODO: understand this part of the code
```

Comment:
The commented-out section is just adjusted according to the updated query, but
I need to understand what this was supposed to accomplish in order to complete the change